src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -60,7 +60,7 @@ class IdentityUserAvatar extends React.Component<Props, State> {
           <hr />
         </div>
         <div className="user-popover-footer">
-          <Link className="user-popover-footer--button" to="/">
+          <Link className="user-popover-footer--button" to="/me/profile">
```

Comment:
This should link directly to the page, rather than going through the deep link, since using a deep link will actually route to the `NotFound` page first, which will also fire an event

Reply:
Got it - thanks. I'll change this to route directly to the page, which is: `/orgs/${org.id}/user/profile`